pkgs/package-overrides.nix

```diff
@@ -95,6 +95,9 @@ in
                   patch "$out" < ${if lib.versionOlder prev.php.version "7.0" then ./intl-icu-patch-5.6-compat.patch else ./intl-icu-patch-7.0-compat.patch}
                 '';
               })
+            ] ++
+            lib.optionals (prev.php.version == "7.2") [
```

Comment:
Again, version is something like `"7.2.34"` so this will not match anything.

Reply:
Indeed ! Will fix accordingly.